okhttp/src/main/kotlin/okhttp3/internal/connection/RealConnectionPool.kt

```diff
@@ -71,7 +71,7 @@ class RealConnectionPool(
   }
 
   /**
-   * Attempts to acquire a recycled connection to [address] for [call]. Returns the connection if it
+   * Attempts to acquire a recycled connection to [address] for [connectionUser]. Returns the connection if it
    * was acquired, or null if no connection was acquired. The acquired connection will also be
    * assigned to [RealCall.connection].
```

Comment:
nit: doesn't always hold. This is through `connectionUser.acquireConnectionNoEvents()` which may no longer be a call.